pkg/karmadactl/interpret/check.go

```diff
@@ -0,0 +1,92 @@
+package interpret
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	"k8s.io/cli-runtime/pkg/printers"
+	"k8s.io/cli-runtime/pkg/resource"
+	cmdutil "k8s.io/kubectl/pkg/cmd/util"
+
+	"github.com/karmada-io/karmada/pkg/resourceinterpreter/configurableinterpreter/luavm"
+)
+
+func (o *Options) runCheck() error {
+	w := printers.GetNewTabWriter(o.Out)
+	defer w.Flush()
+
+	failed := false
+
+	err := o.CustomizationResult.Visit(func(info *resource.Info, _ error) error {
+		var visitErr error
+		fmt.Fprintln(w, "-----------------------------------")
+
+		source := info.Source
+		if info.Name != "" {
+			source = info.Name
+		}
+		fmt.Fprintf(w, "SOURCE: %s\n", source)
+
+		customization, visitErr := asResourceInterpreterCustomization(info.Object)
+		if visitErr != nil {
+			failed = true
+			fmt.Fprintf(w, "%v\n", visitErr)
+			return nil
+		}
+
+		kind := customization.Spec.Target.Kind
+		if kind == "" {
+			failed = true
+			fmt.Fprintln(w, "target.kind no set")
+			return nil
+		}
+		apiVersion := customization.Spec.Target.APIVersion
+		if apiVersion == "" {
+			failed = true
+			fmt.Fprintln(w, "target.apiVersion no set")
+			return nil
+		}
+
+		fmt.Fprintf(w, "TARGET: %s %s\t\n", apiVersion, kind)
+		fmt.Fprintf(w, "RULERS:\n")
+		for _, r := range o.Rules {
+			fmt.Fprintf(w, "    %s:\t", r.Name())
+
+			script := r.GetScript(customization)
+			if script == "" {
+				fmt.Fprintln(w, "DISABLED")
```

Comment:
```suggestion
				fmt.Fprintln(w, "UNSET")
```
I guess `unset` is more accurate than `disabled`.